code/modules/clothing/head/jobs.dm

```diff
@@ -256,7 +270,94 @@
+	icon_state = "beret_ce"
+	armor = list(rad = 20, fire = 30)
+	strip_delay = 60
+
+/obj/item/clothing/head/beret/sci
+	name = "science beret"
+	desc = "A purple beret with the science insignia emblazoned on it. It has that authentic burning plasma smell."
+	icon_state = "beret_sci"
+	armor = list(bomb = 5, bio = 5, fire = 5, acid = 10)
+	strip_delay = 60
+
+//Medical
+/obj/item/clothing/head/beret/med
+	name = "medical beret"
+	desc = "A white beret with a blue cross finely threaded into it. It has that sterile smell about it."
+	icon_state = "beret_med"
+	armor = list(bio = 20)
+	strip_delay = 60
+
+/obj/item/clothing/head/beret/cmo
+	name = "chief medical officer beret"
+	desc = "A baby blue beret with the insignia of Medistan. It smells very sterile."
+	icon_state = "beret_cmo"
+	armor = list(bio = 30, acid = 20)
+	strip_delay = 60
+
+//Centcomm
+/obj/item/clothing/head/beret/cccaptain
+	name = "central command captain beret"
+	desc = "A pure white beret with a Captain insignia of Central Command."
+	icon_state = "beret_centcom_captain"
+	armor = list(melee = 80, bullet = 80, laser = 80, energy = 80, bomb = 80, bio = 80, rad = 80, fire = 80, acid = 80)
+	strip_delay = 120
+
+/obj/item/clothing/head/beret/ccofficer
+	name = "central command officer beret"
+	desc = "A black Central Command Officer beret with matching insignia."
+	icon_state = "beret_centcom_officer"
+	armor = list(melee = 80, bullet = 80, laser = 80, energy = 80, bomb = 80, bio = 80, rad = 80, fire = 80, acid = 80)
+	strip_delay = 120
+
+/obj/item/clothing/head/beret/ccofficernavy
+	name = "central command naval officer beret"
+	desc = "A Navy beret commonly worn by Central Command Naval Officers."
+	icon_state = "beret_centcom_officer_navy"
+	armor = list(melee = 80, bullet = 80, laser = 80, energy = 80, bomb = 80, bio = 80, rad = 80, fire = 80, acid = 80)
+	strip_delay = 120
+
+//For blueshields, but those aren't in so I renamed them to centcomm guards
+/obj/item/clothing/head/beret/ccguard
+	name = "officer beret"
+	desc = "A black Centcom guard's beret."
+	icon_state = "beret_centcom_officer"
+	armor = list(melee = 40, bullet = 20, laser = 10, energy = 10, bomb = 10, bio = 5, rad = 5, fire = 5, acid = 30)
+	strip_delay = 60
+
+/obj/item/clothing/head/beret/ccguardnavy
+	name = "navy officer beret"
+	desc = "A navy Centcomm guard's beret."
+	icon_state = "beret_centcom_officer_navy"
+	armor = list(melee = 40, bullet = 20, laser = 10, energy = 10, bomb = 10, bio = 5, rad = 5, fire = 5, acid = 30)
+	strip_delay = 60
+
+/obj/item/clothing/head/beret/captain
+	name = "captain beret"
+	desc = "A lovely blue Captain beret with a gold and white insignia."
+	icon_state = "beret_captain"
+	armor = list(melee = 50, bullet = 30, laser = 20, energy = 10, bomb = 15, bio = 10, rad = 10, fire = 10, acid = 60)
+	strip_delay = 90
+
 #undef DRILL_DEFAULT
-#undef DRILL_SHOUTING
+#undef DRILL_SHOUTINIG

```

Comment:
Bruh.

Reply:
it was at 3:30 in the morning cut me some slack